### PR TITLE
New endcaps

### PR DESCRIPTION
**Pull request by TiborILD**
BEGINRELEASENOTES
- Bug fixing and modifications in ILD_l2_v02 model 
    - Removed 1st comment line from Hcal_EndcapRing_SD_v01.xml  preventing to appear this detector in the geometry
    -  alligned Hcal_EndcapRing_SD to the back envelope and replaced the material of the remaining free space at the start of this detector by air
    - reduced the thickness of HcalSD_back_plate  from 15->10 mm; gain of additional layer in Endcaps; now at (47/48)
    - ILD_l2_v02  stands for SDHCAL technology in VIDEAU geometry
    - some modifications in Hcal_Endcaps_SD_v01, Hcal_Endcaps_SD_v02
- Added  ILD_l6_v02 for  SDHCAL technology in TESLA geometry
- Added  ILD_s6_v02 a small version w/SDHCAL in TESLA geomtry

ENDRELEASENOTES

### CODE REVIEW

**Comment by TiborILD** [comment]
Hello,
 just wondering that pull request 181 was approved, but this one #180 not yet.
Is taht OK or something went wrong.
thanks
  Tibor

**Comment by gaede** [comment]
@TiborILD 
- couid you please add a first line to the release notes stating that you are updating the ILD - SDHCal models ( lcgeo is also used by CLICdp and SiD)
- your detector naming scheme is somewhat confusing:
      - ILD_l2_v02 had been originally created as 'ILD w/ SDHcal and Videau geometry'
      - this should never have been changed, really
       - I realize that it was accidentally changed in #150 - so it is fine to move back
      - if you now want a model 'ILD w/ SDHcal and Tesla geometry' it should get a new number, e.g. ILD_l6_v02 (assuming it takes all other parameters from the ILD_common_v02)
      - using the o1/o2 identifier to refer to the geometry, whereas in the ILD_ls4/ILD_ls5 models it refers to different readout w/ the same geometry will confuse everyone...

**Comment by TiborILD** [comment]
Frank,
the required changes are done. 
The commit introducing ILD_l2_v02_o2(o1)  is reverted and new model ILD_l6_v02 is added.
Is this OK now as it is?

File: ILD/compact/ILD_l6_v02/ILD_l6_v02.xml [hunk 1]
```diff
@@ -0,0 +1,103 @@
+<lccdd xmlns:compact="http://www.lcsim.org/schemas/compact/1.0"
+       xmlns:xs="http://www.w3.org/2001/XMLSchema"
+       xs:noNamespaceSchemaLocation="http://www.lcsim.org/schemas/compact/1.0/compact.xsd">
+
+  <info name="ILD_l6_v02"
+        title="ILD large model w/ SDHcal in Videau geometry"
```
Comment (gaede) [inline ILD/compact/ILD_l6_v02/ILD_l6_v02.xml]: can you update this to 'Tesla geometry' - and add yourself as author

**Comment by TiborILD** [comment]
OK, good idea; added also ILD_s6_v02

**Comment by gaede** [comment]
Merged as failing test (SiD) seems unrelated.

**Comment by protopopescu** [comment]
Any idea on why the SiD test fails now?

**Pull request merged**